runelite-client/src/main/java/net/runelite/client/plugins/timetracking/farming/PatchImplementation.java

```diff
@@ -2787,4 +2787,6 @@ PatchState forVarbitValue(int value)
 	private final Tab tab;
 
 	private final String name;
+
+	private final boolean requiresHealthCheck;
```

Comment:
```suggestion
	private final boolean healthCheckRequired;
```

or write your own getter idc